pom.xml

```diff
@@ -6599,6 +6601,10 @@
 
     <profile>
       <id>sonar</id>
+      <properties>
+        <failIfNoTests>false</failIfNoTests>
```

Comment:
was it a property set by the job? if so, good idea to pull it into the POM.
is it on a per-module basis? if so, would it be better to explicitly set on the involved modules?